### ISSUE TITLE
Failed to load plugin Text to spech

### ISSUE THREAD

**Issue by guimoal** [muchimi/JoystickGremlinEx#145]
Joystick Gremlin Ex 13.40.14ex (m22)
The action_plugin Text to Speech can not load. The log show this text in system tag;

2024-10-15 21:18:52    WARNING 	Loading action_plugins 'text_to_speech' failed due to: ('unknown voice id %s', 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Speech_OneCore\\Voices\\Tokens\\MSTTS_V110_esES_HelenaM')

**Comment by muchimi** [comment]
This is an interesting error, thanks for reporting.  It looksl ike you have a voice pack that is installed but not recognized by the TTS library.  I'll look at ignoring voices that error out and still allow the plugin to load if one voice didn't.

**Comment by muchimi** [comment]
After researching, there are a few installation issues related to the Spanish voice Helena looking at TTS general errors related to that particular voice set.

The error is reported by the TTS API, so not within GremlinEx as far as I can tell.  I've added some additional verbose mode data to display what voices were detected, and also at runtime is case an error is reported so it will be included in the log.

I've also added a "default" voice if the desired voice doesn't work for some reason.

Because GremlinEx uses the Microsoft TTS API (via a third party library) - it's going to be difficult to diagnose but I would look at perhaps re-installing the Spanish language pack for Windows which includes this voice and see if it corrects the problem.

The code changes will appear in m4 hotfix 6.

**Comment by guimoal** [comment]
Thanks, I love this project.

**Comment by guimoal** [comment]
Your response has given me the idea to change the voice, but I changed it in the windows 11 panel "Time and language > Voice" but I didn't fix anything. Then it occurred to me to go to "Voice Properties" and there was the voice in English, I changed it to the Spanish language (mine) and JGEx starts well and charged the plugin. 
I think it is a very good idea to put a voice selector in the program itself so that the voice that has to read the text is chosen.
Good job. Thanks.
![2024-10-17 21_04_31-NVIDIA GeForce Overlay DT](https://github.com/user-attachments/assets/8b66289d-e32c-429e-ab3c-b3742d45bc27)
![2024-10-17 21_13_02-NVIDIA GeForce Overlay DT](https://github.com/user-attachments/assets/2ad41377-1e8d-4b39-83b5-fbdd106cf686)
![2024-10-17 21_13_16-NVIDIA GeForce Overlay DT](https://github.com/user-attachments/assets/0e22d895-22f4-461d-afbe-05b0a7254ef4)
